lib/cext/ruby.h

```diff
@@ -101,11 +101,11 @@ VALUE rb_tr_hidden_variable_get(VALUE object, const char *name);
 
 // Handles
 
-void *rb_tr_handle_for_managed(void *managed);
-void *rb_tr_handle_for_managed_leaking(void *managed);
-VALUE rb_tr_managed_from_handle_or_null(void *handle);
-void *rb_tr_managed_from_handle(void *handle);
-void rb_tr_release_handle(void *handle);
+void* rb_tr_handle_for_managed(VALUE managed);
+void* rb_tr_handle_for_managed_leaking(VALUE managed);
```

Comment:
It is part of the type, semantically at least.
I want to make it clear these functions convert from native (void*) to managed (VALUE) or the opposite. Of course this is only for the reader as they are typedef of each other.
Having the `*` separated makes it harder to see that in my opinion.